src/ray/raylet/scheduling/cluster_resource_scheduler.cc

```diff
@@ -23,21 +23,14 @@ namespace ray {
 
 using namespace ::ray::raylet_scheduling_policy;
 
-ClusterResourceScheduler::ClusterResourceScheduler()
-    : local_node_id_(scheduling::NodeID::Nil()),
-      is_node_available_fn_([](auto) { return true; }) {
-  cluster_resource_manager_ = std::make_unique<ClusterResourceManager>();
-  NodeResources node_resources;
-  Init(node_resources,
-       /*get_used_object_store_memory=*/nullptr,
-       /*get_pull_manager_at_capacity=*/nullptr);
-}
-
 ClusterResourceScheduler::ClusterResourceScheduler(
     scheduling::NodeID local_node_id,
     const NodeResources &local_node_resources,
-    std::function<bool(scheduling::NodeID)> is_node_available_fn)
-    : local_node_id_(local_node_id), is_node_available_fn_(is_node_available_fn) {
+    std::function<bool(scheduling::NodeID)> is_node_available_fn,
+    bool is_local_available)
+    : local_node_id_(local_node_id),
+      is_node_available_fn_(is_node_available_fn),
+      is_local_available_(is_local_available) {
```

Comment:
maybe name `is_local_available_` to `is_local_node_has_raylet_`, so we don't confuse it with `is_node_available_fn_`

Reply:
Sure, just did.